downstream/modules/platform/proc-settings-gw-additional-options.adoc

```diff
@@ -12,7 +12,7 @@ From the *Platform gateway settings* page, you can configure additional platform
 . Click btn:[Edit platform gateway settings].
 . You can configure the following *Other settings*:
 +
-* *JWT expiration buffer in seconds*: The number of seconds before a JWT token's expiration to revoke from the cache.
+* *Jwt expiration buffer in seconds*: The number of seconds before a JWT token's expiration to revoke from the cache.
```

Comment:
Should this be 'Jwt' just like at the start of the sentence?

Reply:
No, this is one of those cases where it's wrong in the UI. So I've matched the selections to the UI where I talk about the UI element but left it as correct in the text.